### PR TITLE
Fix xsw, calcurse, chktex

### PR DESCRIPTION
**Pull request by vladshablinsky**
calcurse - version update.
xsw, chktex, calcurse - fix formulae according to `brew audit --strict`

### CODE REVIEW

File: Library/Formula/chktex.rb [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
   depends_on :tex
 
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"chktex", "-W"
```
Comment (mikemcquaid) [inline Library/Formula/chktex.rb]: Would be good to write out a `.tex` file here and try and test that? If you `grep` through other formulae you will find similar approaches.

File: Library/Formula/xsw.rb [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+  depends_on "sdl_gfx"
 
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"xsw", "-v"
```
Comment (mikemcquaid) [inline Library/Formula/xsw.rb]: Maybe use `xsw2pdf mypresentation.xsw` and write some sample input to generate a PDF?

Comment (vladshablinsky) [inline Library/Formula/xsw.rb]: @mikemcquaid, something is wrong with `xsw2pdf` - it causes segfault. 
It crashes both when I `brew test` and when I `xsw2pdf test.xsw`. However `xsw test.xsw` works fine.
What should I do? Add these lines
```
  test do
    (testpath/"test.xsw").write <<-EOS.undent
      slide:
        -"Hello" x:52 y:34 align:center
        +"world!"
    EOS
    system bin/"xsw2pdf", "test.xsw"
  end
```
or leave everything as it is?

Comment (mikemcquaid) [inline Library/Formula/xsw.rb]: Maybe add it and see if it works on the bot?

Comment (vladshablinsky) [inline Library/Formula/xsw.rb]: @mikemcquaid fails on the bot too, so I reset changes.

Comment (mikemcquaid) [inline Library/Formula/xsw.rb]: Ok, thanks!

Comment (tdsmith) [inline Library/Formula/xsw.rb]: Is it okay that this is segfaulting?

**Comment by asparagui** [comment]
We always :heart: more tests. :+1:

**Pull request closed without merging**